src/quo2/components/loaders/skeleton/view.cljs

```diff
@@ -2,45 +2,68 @@
-               :index   index
-               :content content
-               :color   color})}]
-    [rn/view
-     {:style (style/content-view
-              {:type    (if (= content :list-items) :author :message)
-               :index   index
-               :content content
-               :color   color})}]
-    (when (= content :notifications)
-      [rn/view
-       {:style (style/content-view {:type    :message2
-                                    :index   index
-                                    :content content
-                                    :color   color})}])]])
+(defn skeleton-item
+  [index content color animated?]
+  [:f>
+   (fn []
+     (let [loading-color           (colors/theme-colors colors/neutral-10 colors/neutral-60)
+           {window-width :width}   (rn/get-window)
+           translate-x             (reanimated/use-shared-value (- window-width))
+           animated-gradient-style (reanimated/apply-animations-to-style
+                                    {:transform [{:translateX translate-x}]}
+                                    {:width  window-width
+                                     :height "100%"})]
+       (reanimated/animate-shared-value-with-repeat translate-x window-width 1000 :linear (- 1) false)
+       [masked-view/masked-view
+        {:style               {:height (get-in constants/layout-dimensions [content :height])}
+         :accessibility-label (if animated? :skeleton-animated :skeleton-static)
+         :maskElement         (reagent/as-element
+                               [rn/view
+                                {:style (style/container content)}
+                                [rn/view
+                                 {:style (style/avatar color)}]
+                                [rn/view {:style style/content-container}
+                                 [rn/view
+                                  {:style (style/content-view
+                                           {:type    (if (= content :list-items) :message :author)
+                                            :index   index
+                                            :content content
+                                            :color   color})}]
+                                 [rn/view
+                                  {:style (style/content-view
+                                           {:type    (if (= content :list-items) :author :message)
+                                            :index   index
+                                            :content content
+                                            :color   color})}]]])}
+        [rn/view
+         {:style {:flex             1
+                  :background-color color}}
+         (when animated?
+           [reanimated/linear-gradient
+            {:colors [color color loading-color color color]
+             :start  {:x 0 :y 0}
+             :end    {:x 1 :y 0}
+             :style  animated-gradient-style}])]]))])
 
 (defn- internal-view
```

Comment:
perhaps this would be better with `content` as a second parameter?
so it can be 
```
(defn- internal-view
  [{:keys [theme blur? parent-height animated?]} content]
  (let [skeleton-height     (get-in constants/layout-dimensions [content :height])
        number-of-skeletons (int (Math/ceil (/ parent-height skeleton-height)))
        color               (cond
                              blur?           colors/white-opa-5
                              (= theme :dark) colors/neutral-90
                              :else           colors/neutral-5)]

    [rn/view {:style {:padding 8}}
     (doall
      (for [index (range number-of-skeletons)]
        ^{:key index}
        [skeleton-item (mod index 4) content color animated?]))]))
        ```
        
        and then used as 
        ```
        [quo/loader {...}
          [... more components]
        ]
        ```

Reply:
if you make this change be sure to update uses across the app :)

Reply:
actually @FFFra please ignore this suggestion of mine :) I thought `content` was like `children` components but I see it is a keyword so best to leave it as is 👍

Reply:
Nice, and tks for telling me. I was about to do that lol